parquet-column/src/main/java/org/apache/parquet/filter2/predicate/FilterApi.java

```diff
@@ -257,6 +258,10 @@ public static <T extends Comparable<T>, C extends Column<T> & SupportsEqNotEq> N
     return new NotIn<>(column, values);
   }
 
+  public static <T extends Comparable<T>> Contains<T> contains(Eq<T> pred) {
```

Comment:
In a fast-follow-up PR we can just replace this signature with:

```java
public static <T extends Comparable<T>> Contains<T> contains(ColumnFilterPredicate<T> pred) {
   ...
}
```

and remove the `SupportsContains` annotation. I did a few tests of `Contains(lt)` and `Contains(gt)` and everything is working as expected 👍

Reply:
understood -- I can commit to working on the follow-up PR right away 👍